optional-container-engine/README.md

```diff
@@ -0,0 +1,73 @@
+# Deploy Bookshelf to Container Engine/Kubernetes
+
+This optional tutorial will walk you through how to deploy the Bookshelf sample application to [Google Container Engine](https://cloud.google.com/container-engine/). This tutorial is also applicable to [Kubernetes](http://kubernetes.io/) outside of Container Engine, but may require additional steps for external load balancing.
+
+## Pre-requisites
+
+1. Create a project in the [Google Cloud Platform Console](https://console.cloud.google.com).
+
+2. [Enable billing](https://console.cloud.google.com/project/_/settings) for your project. 
+
+3. Install the [Google Cloud SDK](https://cloud.google.com/sdk)
+
+        $ curl https://sdk.cloud.google.com | bash 
+        $ gcloud init
+
+4. Install [Docker](https://www.docker.com/).
+
```

Comment:
and pub/sub...though it was default enabled because it is almost all of my projects but just created a new one and it isn't.